.github/workflows/downstream_protobuf_compatibility_check_nightly.yaml

```diff
@@ -6,16 +6,20 @@ on:
   workflow_dispatch:
     inputs:
       protobuf_runtime_versions:
-        description: 'Comma separated list of Protobuf-Java versions (i.e. "3.25.x","4.x.y")'
+        description: 'Comma separated list of Protobuf-Java versions (i.e. "3.25.x","4.x.y").
+                      Note: Surround each version in the list with quotes ("")'
         required: true
   schedule:
     - cron: '0 1 * * *' # Nightly at 1am
 
 name: Downstream Protobuf Compatibility Check Nightly
 jobs:
   downstream-protobuf-test:
-    # Checks if PR comes from Release-Please branch or if invoked from nightly job
-    if: github.head_ref == 'release-please--branches--main' || github.event_name == 'schedule'
+    # This job runs if any of the three conditions match:
+    # 1. PR is raised from Release-Please (PR comes from branch: release-please--branches-{branch_name})
```

Comment:
It is hardcoded below that it only works for main branch's release PRs, can you change one of them to match the description and behavior?

Reply:
Updated